packages/aws-cdk-lib/cx-api/FEATURE_FLAGS.md

```diff
@@ -161,7 +162,8 @@ The following json shows the current recommended set of flags, as `cdk init` wou
     "@aws-cdk/aws-stepfunctions-tasks:fixRunEcsTaskPolicy": true,
     "@aws-cdk/aws-ec2:bastionHostUseAmazonLinux2023ByDefault": true,
     "@aws-cdk/aws-route53-targets:userPoolDomainNameMethodWithoutCustomResource": true,
-    "@aws-cdk/aws-elasticloadbalancingV2:albDualstackWithoutPublicIpv4SecurityGroupRulesDefault": true
```

Comment:
I believe this section got removed by mistake while resolving merge conflict

Reply:
This is not removed but just added a comma at the end.